src/lib/components/Access/EmbargoDateField.js

```diff
@@ -29,15 +29,17 @@ export function EmbargoDateField({
 }
 
 EmbargoDateField.propTypes = {
-  fieldPath: PropTypes.string.isRequired,
-  label: PropTypes.string.isRequired,
-  labelIcon: PropTypes.string.isRequired,
-  placeholder: PropTypes.string.isRequired,
+  fieldPath: PropTypes.string,
+  label: PropTypes.string,
+  labelIcon: PropTypes.string,
+  placeholder: PropTypes.string,
+  required: PropTypes.bool,
 };
 
 EmbargoDateField.defaultProps = {
-  fieldPath: 'access.embargo.until',
-  label: i18next.t('Embargo until'),
-  labelIcon: 'calendar',
-  placeholder: i18next.t('YYYY-MM-DD'),
+  required: false,
+  labelIcon: "calendar",
+  placeholder: i18next.t("YYYY-MM-DD"),
+  fieldPath: "access.embargo.until",
```

Comment:
if we upper pin invenio-app-rdm and we release these changes as `v0.20.x` then we should remove the `defaultProp` `fieldPath`...otherwise, you need here an ignore rule no? I would also maybe add a comment saying that this will be removed in the future or similar...

Reply:
Please take this into account for all the `eslint` ignore rules you added :)

Reply:
Yes, i looked over this one, i just made sure that every `fieldPath` prop is set to `isRequired`!

Reply:
maybe i should post a comment in this PR documenting why this decision (ignoring the fieldPaths defaultProps) was made and link it with every ignore?

Reply:
I would start an issue to clean it out in the 0.20 maybe

Reply:
issue made here https://github.com/inveniosoftware/react-invenio-deposit/issues/563